docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,200 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'node-cluster-rsrc-use.json':
+      local legendLink = '%s/dashboard/file/k8s-node-rsrc-use.json' % $._config.grafana_prefix;
+
+      g.dashboard('USE Method / Cluster')
+      .addRow(
+        g.row('CPU')
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:avg_rate1m
```

Comment:
This is missing the node selector, plus the ones below

Reply:
Right, with my rule changes, I can add it everywhere ow.